js/main.js

```diff
@@ -22,4 +32,92 @@ $form.addEventListener('submit', function (event) {
   data.nextEntryId++;
 
   $form.reset();
+
+  const $newDOM = renderEntry(obj);
+
+  $ul.prepend($newDOM);
+
+  viewSwap('entries');
+
+  if (data.entries.length <= 0) {
+    toggleNoEntrires();
+  }
```

Comment:
This `if` logic is already inside of `toggleNoEntries`, so you should just call `toggleNoEntries` directly